README.release_notes

```diff
@@ -0,0 +1,21 @@
+@JuliaRegistrator register
+
+Release notes:
+
+Note that this release increases the minor version and contains a number of breaking changes:
+
+- SCF solver implementations have changed their API from `ρout = F(ρin)` to `ρout, info_out = F(ρin, info_in)`. This means that many of DFTK's learning resources (which use this feature to introduce the concepts of basic SCF algorithms) need to be adapted for DFTK 0.7. (#811)
+
+- The `model_LDA`, `model_PBE` and `model_SCAN` functions are now deprecated and will be removed in DFTK version 0.7.1. Use `model_DFT(system; functionals=LDA(), kwargs...)` or similar (`functionals=PBE()`, `functionals=SCAN()`) (#994).
```

Comment:
using the `LDA()`, `PBE()` and similar things is what we need for making the `AtomsCalculator` interface easier (the alternative is forcibly writing out `[:lda_x, :lda_c_pw]` each time, which people do not remember. Dropping `model_LDA` is then to avoid fragmentation of interfaces that all need to be supported.